lbrynet/tests/unit/daemon/test_claims_comparator.py

```diff
@@ -0,0 +1,27 @@
+import json
+import unittest
+
+from lbrynet.daemon.claims_comparator import arrange_results
+
+
+class ClaimsComparatorTest(unittest.TestCase):
+    def setUp(self):
+        with open('claims_comparator_cases.json') as f:
+            document = json.load(f)
+        self.cases = document['cases']
+
+    def test_arrange_results(self):
+        for case in self.cases:
+            results = case['results']
+            data = {'result': results}
+            expected = case['expected']
+
+            claims = arrange_results([data])
+            claim = claims[0]
+            actual = claim['result']
+
+            self.assertEqual(expected, actual, case['description'])
```

Comment:
A loop like this makes it impossible to set breakpoints on a specific test case. Can you refactor this so that there is one line per test case which can accept a breakpoint in debugger if necessary. Even better if each test case is its own method. If there are 4 use cases then there should be 4 methods (descriptively named explaining what is being tested).